compiler/src/formatting/format.re

```diff
@@ -3672,22 +3684,11 @@ let rec print_data =
             print_type(~original_source, ~comments, t);
           };
 
-          let after_angle_comments =
-            Comment_utils.get_after_brace_comments(
-              ~loc=get_loc(first),
-              comments,
-            );
-          let cleaned_comments =
-            remove_used_comments(
-              ~remove_comments=after_angle_comments,
-              comments,
-            );
-
           let param_items =
             item_iterator(
               ~get_loc,
               ~print_item,
-              ~comments=cleaned_comments,
+              ~comments=[],
```

Comment:
Does this mean that comments inside parameterized types are no longer support? Do we have a test for something like `type Foo<A /* something */, B> = (A, B)`?

Reply:
This is broken, but not by this change, so I'll fix

Reply:
Fixed in #1460